apis/quota/v1alpha1/elastic_quota_profile_types.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2022 The Koordinator Authors.
```

Comment:
All new files should be `2023`

Reply:
the license-check will check it and reject it